### PR TITLE
Fix for incorrect planesremoved events (firing ids instead of planes)

### PR DESCRIPTION
**Pull request by lincolnfrog**

### CODE REVIEW

File: WebARonARKit/WebARonARKit.js [hunk 1]
```diff
@@ -882,7 +882,8 @@
       }
     } else if (event.type == "planesremoved") {
       for (var i = 0; i < event.planes.length; i++) {
-        WebARonARKitVRDisplay.planes_.delete(event.planes[i]);
+        var plane = event.planes[i];
```
Comment (judax) [inline WebARonARKit/WebARonARKit.js]: It is a comment from a previous change but I think events should be fired after the internal structure is changed, not after (referring to the dispatchEvent call at the beginning on this function.

File: WebARonARKit/WebARonARKit/ViewController.m [hunk 2]
```diff
@@ -608,24 +608,7 @@ - (void)session:(ARSession *)session didUpdateAnchors:(nonnull NSArray<ARAnchor
...
-                           (int)plane.identifier];
-    if (i < anchors.count - 1) {
-      identifierStr = [identifierStr stringByAppendingString:@","];
-    }
-    removedStr = [removedStr stringByAppendingString:identifierStr];
-  }
-  removedStr = [removedStr stringByAppendingString:@"]"];
-
-  [self dispatchVRDisplayPlaneEvent:@"planesremoved" planes:removedStr];
+  [self dispatchVRDisplayPlaneEvent:@"planesremoved" planes:[self getPlanesString:anchors]];
```
Comment (judax) [inline WebARonARKit/WebARonARKit/ViewController.m]: What is the purpose of this change? To reuse as much code as possible? Think that the structure that needs to be created is more complex with unnecessary data (matrices). In ARCore, the same approach (ids only) is being used right?

Comment (lincolnfrog) [reply WebARonARKit/WebARonARKit/ViewController.m]: It means I don't have to intercept the event in javascript and then rebuild a new event, which seems very bizarre to me. I think this is the cleanest solution.

File: WebARonARKit/WebARonARKit.js (hunk 1 above)
Comment (lincolnfrog) [reply WebARonARKit/WebARonARKit.js]: done.
